ruby-challenges.rb

```diff
@@ -32,3 +55,18 @@
 # Expected output: 'LEARN is not a palindrome'
 palindrome_tester3 = 'Rotator'
 # Expected output: 'Rotator is a palindrome'
+
+
+def palindrome string
+    if string == string.reverse
+        return "#{string} is a palindrome"
```

Comment:
Nice work in this section!  Just remember that Ruby uses implicit returns, so there is no need to return inside your conditionals.